src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
HTML Content: Consider using semantic tags instead of divs. For example, you could use the `section` tag to group related content. This will improve the SEO of your page and help screen readers understand the content better.